Applications/Spire/Include/Spire/SecurityInput/SecurityInfoWidget.hpp

```diff
@@ -1,25 +1,16 @@
 #ifndef SPIRE_SECURITY_INFO_WIDGET_HPP
 #define SPIRE_SECURITY_INFO_WIDGET_HPP
 #include <QLabel>
-#include <QWidget>
 #include "Nexus/Definitions/SecurityInfo.hpp"
 #include "Spire/SecurityInput/SecurityInput.hpp"
+#include "Spire/Ui/DropDownItem.hpp"
 
 namespace Spire {
 
   //! A convenience widget to display a security's name and ticker symbol.
-  class SecurityInfoWidget : public QWidget {
+  class SecurityInfoWidget : public DropDownItem {
```

Comment:
Sticking to the theme of using descriptive names, rename this to ```SecurityInfoItem```. Let's do our best to avoid using generic terms like ```Widget``` unless it truly is a widget that is dissimilar to anything else.